### ISSUE TITLE
Current ctx is no longer indicating the current context with kctx to list the contexts

### ISSUE THREAD

**Issue by ye** [ahmetb/kubectx#144]
This used to work just as the front page text gif demo'd when I was using Bash, but after switching to oh my zsh. What it brings up is a selector style menu but it doesn't indicate the current context, as seen below, it always has the cursor on `ftp` context :(

![Screen Shot 2019-04-15 at 4 28 32 PM](https://user-images.githubusercontent.com/133924/56163341-11391000-5f9c-11e9-82b8-345dca923004.png)

So I am wondering what caused it to have this behavior or how do I fix it?

**Comment by ahmetb** [comment]
> What it brings up is a selector style menu

I'm assuming this behavior is consistent in both bash/zsh, it just looks at whether `fzf` is installed.

> but it doesn't indicate the current context

what version are you on? v0.6.3 has fixed this: https://github.com/ahmetb/kubectx/releases

**Comment by ye** [comment]
> I'm assuming this behavior is consistent in both bash/zsh, it just looks at whether fzf is installed.
Yes, it is! I realized that I did just recently installed `fzf` plugin and behaviors on both shells are consistent. Thanks for the tip!

> what version are you on? v0.6.3 has fixed this: https://github.com/ahmetb/kubectx/releases
I was on v0.6.2 and since upgrade to v0.6.3, and yes it does seem to highlight the current context in a different color (a little bit harder to discern though I guess I can tweak _KUBECTX_FORCE_COLOR env variable to highlight is a brighter vivid color of yellow?) Eitherway, thank you!
